umbral/utils.py

```diff
@@ -42,3 +44,23 @@ def kdf(ecpoint, key_length):
 
 def get_curve_keysize_bytes(curve):
     return int(math.ceil(curve.key_size / 8.00))
+
+
+class AbstractFromToBytes(ABC):
+    @staticmethod
+    @abstractmethod
+    def from_bytes(cls, data: bytes, curve: ec.EllipticCurve = None):
+        """
+        Instantiate a KFrag object from the serialized data.
```

Comment:
The comments on this file should be generic

Reply:
Would it work as `Instantiate a cls object from the serialised data.`?

Reply:
Over correction, IMO. How about referring to "split-key fragment object".